public/scripts/ordering.js

```diff
@@ -10,10 +10,10 @@ const CLASS_ORDER_ALPHABETICALLY = 'order-alphabetically';
 const CLASS_ORDER_BY_COLOR = 'order-by-color';
 const CLASS_ORDER_BY_RELEVANCE = 'order-by-relevance';
 
-let activeOrdering = DEFAULT_ORDERING;
-let preferredOrdering = DEFAULT_ORDERING;
-
 export default function initOrdering(document, storage) {
+  let activeOrdering = null;
+  let preferredOrdering = null;
```

Comment:
Should these be `null` or `DEFAULT_ORDERING`. The latter has the main advantage that nothing will be stored in local storage until the user has actually chosen an ordering.

Reply:
They have both been set to `DEFAULT_ORDERING` as per the advantage above. Additionally, the DEFAULT_ORDERING is no longer explicitly loaded [later in the file](https://github.com/simple-icons/simple-icons-website/blob/6bcf096af11224ab73b36358130e76575266b2ac/public/scripts/ordering.js#L30) either, for the same reason.